canonical/ubuntu.com
# Feature rebrand IoT

## Done

- Feature branch for the IoT bubble rebrand

## QA

- View the site locally in your web browser at: https://ubuntu-com-14539.demos.haus/internet-of-things
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check bubble throughout 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16362
